UPGRADING.rst

```diff
@@ -10,6 +10,12 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2025 Register indexer SA with Broad's SAM during deployment
```

Comment:
Double check if this is really where you introduced these variables.

Reply:
https://github.com/DataBiosphere/azul/commit/08e765f76f38c9764d4dee912b146d7861b30c5d#diff-ca6bc615c166eaf90c362521c5bdab0cR37-R38